src/main/java/frc/robot/subsystems/turretSubsystem/TurretCommandFactory.java

```diff
@@ -89,8 +125,7 @@ public Command zeroZeroing() {
               highGearCANcoderConfigurator.apply(newCfgHighGear);
             },
             subsystem)
-        .withName("zeroZeroing")
-        .asProxy();
+        .withName("zeroZeroing");
```

Comment:
For my own edification: why is it called this?

Reply:
It configures the zero position to be the current position.